docs: remove deprecated `@stable` jsdoc tags

petebacondarwin opened the pull request.

petebacondarwin committed:
docs: remove all deprecated `@stable` jsdoc tags

These are no longer needed as stable docs are computed as those that
do not have `@experimental` or `@deprecated` tags.

petebacondarwin committed:
docs(animations): migrate deprecated `@whatItDoes` tag

This was not being picked up by the doc-gen as it does not appear to be
part of the public API.

mary-poppins commented:
You can preview 4a4d633 at https://pr23210-4a4d633.ngbuilds.io/.

IgorMinar commented:
@petebacondarwin all the golden files for api tests need to be updated. can you please do that? thanks

petebacondarwin committed:
fixup! docs: remove all deprecated `@stable` jsdoc tags

These are no longer needed as stable docs are computed as those that
do not have `@experimental` or `@deprecated` tags.

mary-poppins commented:
You can preview 2503163 at https://pr23210-2503163.ngbuilds.io/.

mary-poppins commented:
You can preview 572e027 at https://pr23210-572e027.ngbuilds.io/.

mary-poppins commented:
You can preview 002e714 at https://pr23210-002e714.ngbuilds.io/.
